gradle.properties

```diff
@@ -17,3 +17,4 @@ org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
 android.useAndroidX=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+# ID query parameters to call the API
```

Comment:
this comment is not necessary anymore, please delete it

Reply:
Done